core-api/src/main/java/com/optimizely/ab/optimizelyconfig/OptimizelyConfigService.java

```diff
@@ -68,28 +70,36 @@ String getExperimentFeatureKey(String experimentId) {
     @VisibleForTesting
     Map<String, OptimizelyExperiment> getExperimentsMap() {
         List<Experiment> experiments = projectConfig.getExperiments();
-        if(experiments == null) {
+        if (experiments == null) {
             return Collections.emptyMap();
         }
-        return experiments.stream().collect(Collectors.toMap(Experiment::getKey, experiment -> new OptimizelyExperiment(
-            experiment.getId(),
-            experiment.getKey(),
-            getVariationsMap(experiment.getVariations(), experiment.getId())
-        )));
+        Map<String, OptimizelyExperiment> featureExperimentMap = new HashMap<>();
+        for (Experiment experiment : experiments) {
+            featureExperimentMap.put(experiment.getKey(), new OptimizelyExperiment(
+                experiment.getId(),
+                experiment.getKey(),
+                getVariationsMap(experiment.getVariations(), experiment.getId())
+            ));
+        }
+        return featureExperimentMap;
```

Comment:
"featureExperimentMap" can mislead. Change it to "experimentsMap" or "experimentKeyMap"